overlay/packages/apps/CarrierConfig/res/xml/vendor.xml

```diff
@@ -623,23 +623,35 @@
     </carrier_config>
     <carrier_config mcc="286" mnc="01">
         <boolean name="carrier_volte_available_bool" value="true" />
-        <boolean name="carrier_volte_roaming_available_bool" value="false" />
+        <boolean name="carrier_wfc_ims_available_bool" value="true" />
+        <boolean name="carrier_default_wfc_ims_enabled_bool" value="true" />
         <int name="carrier_default_wfc_ims_mode_int" value="1" />
-        <int name="carrier_default_wfc_ims_roaming_mode_int" value="1" />
+        <int name="carrier_default_wfc_ims_roaming_mode_int" value="2" />
```

Comment:
Why the change of mode here ?

Reply:
value=2 indicates to wifi preferred in roaming mode rather than cellular mode
also stock's default value and turkish operators knows own people :) and it's more safe when someone using in roaming mode

Reply:
Im aware of what it means. I was mostly asking why was the change of the mode changed but since its from stock we can assume its safe from the looks of it.